source/quick-start.txt

```diff
@@ -14,6 +14,49 @@ Quick Start
 
 .. include:: /includes/quick-start/overview.rst
 
+Create a MongoDB Cluster
+------------------------
+
+Set Up a Free Tier Cluster in Atlas
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After setting up your .NET project dependencies, create a MongoDB cluster
+where you can store and manage your data. Complete the
+:atlas:`Get Started with Atlas </getting-started>` guide to set up a new
+Atlas account and free tier MongoDB cluster, load datasets, and
+interact with the data.
+
+After completing the steps in the Atlas guide, you have a new MongoDB
+cluster deployed in Atlas, a new database user, and
+:atlas:`sample datasets loaded </sample-data/>` into your cluster.
+
+Find Your Connection String
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To connect to the Atlas cluster you created in the previous step, you need the 
+*connection string* for your cluster and user. A connection string contains the 
+information the {+driver-short+} needs to connect 
+to a MongoDB cluster, including the hostname or IP address and port of the
+cluster, authentication mechanism, user credentials (when applicable), and
+other connection options.
+
+To retrieve the connection string for your Atlas cluster and user, complete the 
+:guides:`Get Connection String </atlas/connection-string>`
+guide. Once you have finished the guide, you will have a connection string similar to the 
+following:
+
+.. code-block:: bash
+
+   "mongodb+srv://username:password@cluster0.abc.mongodb.net/?retryWrites=true&w=majority"
+
+Save your Atlas connection string to an environment variable named ``MONGODB_URI``. 
```

Comment:
Moved this step to immediately after they retrieve the string, rather than having them copy it down somewhere else first.

Reply:
I think might be confusing here for some users since they haven't created the app or the project directory yet. If I were starting off from scratch and reading this I wouldn't know if this should be in the same location as the app or somewhere else entirely.
 
I also wonder if all users will know what exactly we mean by saying to "save to an environment variable", but that might be a separate discussion.

Reply:
Yeah, the "environment variable" language was already here, but we can talk about whether to clarify it. 

I don't think it makes a difference where they run this command, since it's an environment-wide variable. What if we explain the steps a little more?